underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -477,7 +477,8 @@ private boolean isRoot(String key) {
 
   /**
    * Lists the files in the given path, the paths will be their logical names and not contain the
-   * folder suffix.
+   * folder suffix. Note that, due to the limitation of OSS client, this method can only return up
+   * to 1000 objects.
```

Comment:
@yupeng9 could you confirm with Alibaba that there is no way to list more than 1000 objects? thanks